opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -859,11 +1193,24 @@ mod tests {
         assert_eq!(1, exporter.get_emitted_logs().unwrap().len())
     }
 
+    #[tokio::test(flavor = "current_thread")]
+    async fn test_batch_log_processor_shutdown_under_async_runtime_current_flavor_multi_thread() {
+        let exporter = InMemoryLogExporterBuilder::default().build();
+        let processor = BatchLogProcessor::new(Box::new(exporter.clone()), BatchConfig::default());
+
+        //
+        // deadloack happens in shutdown with tokio current_thread runtime
```

Comment:
is this true with new implementation?

Reply:
Removed the comments. It doesn't apply and the test is also enabled for the new implementation to make sure it doesn't happen.